src/utils/hints/microblocksHints.js

```diff
@@ -0,0 +1,6 @@
+export const microblocksHints = {
+  transactionHash: 'Unique identifier of the transaction.',
+  transactionTime: 'Date and time when the transaction was created.',
```

Comment:
```suggestion
  transactionTime: 'Keyblock height and exact date and time when the transaction has been executed by inclusion into a microblock.',
```

I would use the same we have in other places.

Reply:
Right, fixed